typescript/introduction/tsc-basics/tsc-classes-intro.md

```diff
@@ -0,0 +1,62 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Classes Intro
+
+---
+## Content
+
+If you're not familiar with classes, these represent the building blocks of the object-oriented class-based programming approach. TypeScript allows you to use classes, and when the code is compiled it is compatbile with all the major broswers and platofrms. In this insight we will take a look at a simple class, with more in-depth information being given in a future mission.
+
+So, what is a class? To best explain it, let's take a look at an example:
```

Comment:
```suggestion
Classes represent the building blocks of the style of programming called object-oriented (class-based) programming, or OOP[1]. 

The idea behind it is to organize sections of the code into definitions of entities that exist in your program (an "Animal", a "GamePlayer", a "User", etc). Then you can use these definitions, called classes, to create (instantiate) those entities wherever you'd like[1].

To achieve this, TypeScript gives us the `class` keyword:
```

[1] An instantiate class is called an object. This is the first O in OOP comes from.